From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

bwren committed:
updates

bwren committed:
updates

bwren committed:
records

bwren committed:
link fixes

bwren committed:
lionk fix

HeidiSteen committed:
Azure Search: Concurrency article and TOC entry

burkeholland committed:
Update documentdb-nodejs-application.md

Remove section on styling (not needed) and update form to use Bootstrap forms.

bwren committed:
updates

bwren committed:
for review

v-viinde committed:
USER STORY 1037362 SaaS App Tutorial: Peoplecart

v-viinde committed:
Updated

swkrish committed:
Merge pull request #4 from MicrosoftDocs/master

Update private branch

v-viinde committed:
Updated

swkrish committed:
Merge pull request #5 from MicrosoftDocs/master

update private branch

swkrish committed:
added rollover instructions

swkrish committed:
minor edits

swkrish committed:
added rollover faq

swkrish committed:
added note about accidental extra rollover

swkrish committed:
added Edge and Safari support

swkrish committed:
added note about Safari

swkrish committed:
added limitations

swkrish committed:
Update active-directory-aadconnect-sso-quick-start.md

swkrish committed:
added step about machine account rollover

swkrish committed:
fixed broken link

swkrish committed:
edits

jpallavi committed:
Create backup-azure-log-analytics-data-model.md

jpallavi committed:
Update backup-azure-log-analytics-data-model.md

jpallavi committed:
Update backup-azure-log-analytics-data-model.md

jpallavi committed:
Update backup-azure-log-analytics-data-model.md

jpallavi committed:
Update backup-azure-log-analytics-data-model.md

genlin committed:
add new article

genlin committed:
correct typo

genlin committed:
correct typo

genlin committed:
correct link issue

jpallavi committed:
Update backup-azure-log-analytics-data-model.md

HeidiSteen committed:
TOC change

HeidiSteen committed:
revised doc to match the two code samples

HeidiSteen committed:
edits to section headers, presentation

HeidiSteen committed:
another round of formatting edits

jpallavi committed:
Update backup-azure-log-analytics-data-model.md

jpallavi committed:
Update backup-azure-log-analytics-data-model.md

jpallavi committed:
Update backup-azure-log-analytics-data-model.md

jpallavi committed:
Update backup-azure-log-analytics-data-model.md

jpallavi committed:
Update backup-azure-log-analytics-data-model.md

genlin committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 61799

genlin committed:
update the file name

genlin committed:
update toc

genlin committed:
update TOC

genlin committed:
correct the format issue

genlin committed:
add shell files

genlin committed:
add toc

genlin committed:
correct typo

genlin committed:
rewrite a step

genlin committed:
rewording a steps

genlin committed:
correct format issue

jpallavi committed:
Update backup-azure-configure-reports.md

HeidiSteen committed:
typo

HeidiSteen committed:
removed the typo accidentally introduced with the last fix

mmacy committed:
Moved articles\container-service\container-service-deploy-spring-boot-app-on-kubernetes.md to articles\container-service\kubernetes\container-service-deploy-spring-boot-app-on-kubernetes.md

mmacy committed:
Moved articles\container-service\container-service-deploy-spring-boot-app-on-linux.md to articles\container-service\kubernetes\container-service-deploy-spring-boot-app-on-linux.md

mmacy committed:
Moved articles\container-service\container-service-draft-up.md to articles\container-service\kubernetes\container-service-draft-up.md

mmacy committed:
Moved articles\container-service\container-service-kubernetes-coscale.md to articles\container-service\kubernetes\container-service-kubernetes-coscale.md

mmacy committed:
Moved articles\container-service\container-service-kubernetes-datadog.md to articles\container-service\kubernetes\container-service-kubernetes-datadog.md

mmacy committed:
Moved articles\container-service\container-service-kubernetes-helm.md to articles\container-service\kubernetes\container-service-kubernetes-helm.md

mmacy committed:
Moved articles\container-service\container-service-kubernetes-jenkins.md to articles\container-service\kubernetes\container-service-kubernetes-jenkins.md

mmacy committed:
Moved articles\container-service\container-service-kubernetes-load-balancing.md to articles\container-service\kubernetes\container-service-kubernetes-load-balancing.md

mmacy committed:
Moved articles\container-service\container-service-kubernetes-oms.md to articles\container-service\kubernetes\container-service-kubernetes-oms.md

mmacy committed:
Moved articles\container-service\container-service-kubernetes-service-principal.md to articles\container-service\kubernetes\container-service-kubernetes-service-principal.md

mmacy committed:
Moved articles\container-service\container-service-kubernetes-sysdig.md to articles\container-service\kubernetes\container-service-kubernetes-sysdig.md

mmacy committed:
Moved articles\container-service\container-service-kubernetes-ui.md to articles\container-service\kubernetes\container-service-kubernetes-ui.md

mmacy committed:
Moved articles\container-service\container-service-kubernetes-walkthrough.md to articles\container-service\kubernetes\container-service-kubernetes-walkthrough.md

mmacy committed:
Moved articles\container-service\container-service-kubernetes-windows-walkthrough.md to articles\container-service\kubernetes\container-service-kubernetes-windows-walkthrough.md

mmacy committed:
Moved articles\container-service\container-service-tutorial-kubernetes-app-update.md to articles\container-service\kubernetes\container-service-tutorial-kubernetes-app-update.md

mmacy committed:
Moved articles\container-service\container-service-tutorial-kubernetes-deploy-application.md to articles\container-service\kubernetes\container-service-tutorial-kubernetes-deploy-application.md

mmacy committed:
Moved articles\container-service\container-service-tutorial-kubernetes-deploy-cluster.md to articles\container-service\kubernetes\container-service-tutorial-kubernetes-deploy-cluster.md

mmacy committed:
Moved articles\container-service\container-service-tutorial-kubernetes-monitor.md to articles\container-service\kubernetes\container-service-tutorial-kubernetes-monitor.md

mmacy committed:
Moved articles\container-service\container-service-tutorial-kubernetes-prepare-acr.md to articles\container-service\kubernetes\container-service-tutorial-kubernetes-prepare-acr.md

mmacy committed:
Moved articles\container-service\container-service-tutorial-kubernetes-prepare-app.md to articles\container-service\kubernetes\container-service-tutorial-kubernetes-prepare-app.md

mmacy committed:
Moved articles\container-service\container-service-tutorial-kubernetes-scale.md to articles\container-service\kubernetes\container-service-tutorial-kubernetes-scale.md

mmacy committed:
Moved articles\container-service\container-service-application-specific-marathon.md to articles\container-service\dcos-swarm\container-service-application-specific-marathon.md

mmacy committed:
Moved articles\container-service\container-service-create-acs-cluster-cli.md to articles\container-service\dcos-swarm\container-service-create-acs-cluster-cli.md

mmacy committed:
Moved articles\container-service\container-service-dcos-acr.md to articles\container-service\dcos-swarm\container-service-dcos-acr.md

mmacy committed:
Moved articles\container-service\container-service-dcos-agents.md to articles\container-service\dcos-swarm\container-service-dcos-agents.md

mmacy committed:
Moved articles\container-service\container-service-dcos-fileshare.md to articles\container-service\dcos-swarm\container-service-dcos-fileshare.md

mmacy committed:
Moved articles\container-service\container-service-dcos-manage-tutorial.md to articles\container-service\dcos-swarm\container-service-dcos-manage-tutorial.md

mmacy committed:
Moved articles\container-service\container-service-dcos-quickstart.md to articles\container-service\dcos-swarm\container-service-dcos-quickstart.md

mmacy committed:
Moved articles\container-service\container-service-dcos-vamp-canary-release.md to articles\container-service\dcos-swarm\container-service-dcos-vamp-canary-release.md

mmacy committed:
Moved articles\container-service\container-service-deployment.md to articles\container-service\dcos-swarm\container-service-deployment.md

mmacy committed:
Moved articles\container-service\container-service-docker-swarm-mode-setup-ci-cd-acs-engine.md to articles\container-service\dcos-swarm\container-service-docker-swarm-mode-setup-ci-cd-acs-engine.md

mmacy committed:
Moved articles\container-service\container-service-docker-swarm-setup-ci-cd.md to articles\container-service\dcos-swarm\container-service-docker-swarm-setup-ci-cd.md

mmacy committed:
Moved articles\container-service\container-service-docker-swarm.md to articles\container-service\dcos-swarm\container-service-docker-swarm.md

mmacy committed:
Moved articles\container-service\container-service-enable-public-access.md to articles\container-service\dcos-swarm\container-service-enable-public-access.md

mmacy committed:
Moved articles\container-service\container-service-load-balancing.md to articles\container-service\dcos-swarm\container-service-load-balancing.md

mmacy committed:
Moved articles\container-service\container-service-mesos-marathon-rest.md to articles\container-service\dcos-swarm\container-service-mesos-marathon-rest.md

mmacy committed:
Moved articles\container-service\container-service-mesos-marathon-ui.md to articles\container-service\dcos-swarm\container-service-mesos-marathon-ui.md

mmacy committed:
Moved articles\container-service\container-service-monitoring-dynatrace.md to articles\container-service\dcos-swarm\container-service-monitoring-dynatrace.md

mmacy committed:
Moved articles\container-service\container-service-monitoring-elk.md to articles\container-service\dcos-swarm\container-service-monitoring-elk.md

mmacy committed:
Moved articles\container-service\container-service-monitoring-oms.md to articles\container-service\dcos-swarm\container-service-monitoring-oms.md

mmacy committed:
Moved articles\container-service\container-service-monitoring-sysdig.md to articles\container-service\dcos-swarm\container-service-monitoring-sysdig.md

mmacy committed:
Moved articles\container-service\container-service-monitoring.md to articles\container-service\dcos-swarm\container-service-monitoring.md

mmacy committed:
[acs] initial commit of external files (non-too-moved)

mmacy committed:
[acs] fix broken link warnings

mmacy committed:
[acs] fix icon missing

mmacy committed:
[acs] breadcrumb fix

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-concurrency

mmacy committed:
[acs] move /scripts to /kubernetes, remove Samples from DCOS TOC

mmacy committed:
[acs] fix sample link errors

HeidiSteen committed:
:wq

Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-concurrency

HeidiSteen committed:
Bruce's PR feedback

mmacy committed:
[acs] redirect hack for Security + redirects for security & cli-samples.md

mmacy committed:
[acs] TOC update (root node, security node)

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-concurrency

HeidiSteen committed:
attempting to resize code block to use less scrolling

HeidiSteen committed:
more retrofitting

mmacy committed:
[acs] include hack for shared articles (-connect, -faq, -scale)

mmacy committed:
[acs] add new K8s intro + move dcos intro to dcos

mmacy committed:
[acs] add new K8s intro + move dcos intro to dcos

mmacy committed:
[acs] update CLI link in K8s intro to point to K8s Quickstart

HeidiSteen committed:
more links

mmacy committed:
[acs] fix overview TOC links

mmacy committed:
[acs] win toc move + remove dcos from samples

nitinme committed:
testing something

johnkemnetz committed:
adding schema info

johnkemnetz committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into activity-log-schema

nitinme committed:
another one

nitinme committed:
Revert "another one"

This reverts commit b51484b00def06b4b88ec616894ffc102377e23b.

nitinme committed:
Revert "testing something"

This reverts commit 80129d0f71c0b4835db67192bec23a23c0b2f1ed.

johnkemnetz committed:
Update monitoring-overview-activity-logs.md

johnkemnetz committed:
Update monitoring-activity-log-schema.md

genlin committed:
apply edit changes

jpallavi committed:
Update backup-azure-log-analytics-data-model.md

jpallavi committed:
Update backup-azure-log-analytics-data-model.md

jpallavi committed:
Update TOC.md

jpallavi committed:
Update backup-azure-configure-reports.md

mmacy committed:
[acs] convert TOC to yml

mmacy committed:
[acs] convert TOC to yml

neilpeterson committed:
updated quickstart

mmacy committed:
Merge pull request #2 from neilpeterson/17904

updated quickstart

neilpeterson committed:
tutorial updates

mmacy committed:
Merge pull request #3 from neilpeterson/nepetes-acs-updates

tutorial updates

neilpeterson committed:
fixing paths

mmacy committed:
Merge pull request #5 from neilpeterson/nepetes-acs-updates

fixing paths

mmacy committed:
[acs] convert /container-service/index.md to *.yml

mmacy committed:
Merge branch 'acs-kube-split-dcosswarm-02' of https://github.com/mmacy/azure-docs-pr into acs-kube-split-dcosswarm-02

neilpeterson committed:
date bump

mmacy committed:
Merge pull request #6 from neilpeterson/nepetes-acs-updates

date bump

mmacy committed:
[acs] fix INCLUDE error and 3 invalid link warnings

mmacy committed:
[acs] fix landing page + collapse Resources in TOC

mmacy committed:
[acs] fix landing page (add blank title: line)

mumian committed:
test the streaming application

swkrish committed:
fixed numbering

swkrish committed:
fixed typo

swkrish committed:
improved readability of new step

johnkemnetz committed:
Update monitoring-activity-log-schema.md

johnkemnetz committed:
Update TOC.md

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-concurrency

HeidiSteen committed:
corrections per feedback, minus code formatting

steveesp committed:
LIS4.2.2-2

steveesp committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into vnet-optimal

LIS4.2.2-2

swkrish committed:
fixed header

swkrish committed:
fixed steps

swkrish committed:
Update active-directory-aadconnect-sso-faq.md

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-happy-path

neilpeterson committed:
standardizing with ACI

WilliamAntonRohm committed:
Merge pull request #18 from MicrosoftDocs/master

getting newest stuff

MightyPen committed:
Clarify columnstore ix merely maintained at Standard tier

Email received from C.R. 2017-07-20 14:59, "FW: Azure SQL DB -...", showed customer or field person misunderstood how Columnstore Indexes interact with Azure SQL Database pricing tiers below Premium.

nitinme committed:
Merge branch 'nitinme-hdi-happy-path' of https://github.com/nitinme/azure-docs-pr into nitinme-hdi-happy-path

mmacy committed:
Merge pull request #8 from neilpeterson/nepetes-acs-updates

standardizing with ACI

WilliamAntonRohm committed:
added "ADLA Quota Limits" article entry

burkeholland committed:
Merge branch 'master' into patch-8

swkrish committed:
Update active-directory-aadconnect-sso-faq.md

WilliamAntonRohm committed:
fixed heading capitalization

burkeholland committed:
Update documentdb-nodejs-application.md

neilpeterson committed:
copy update

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-concurrency

nitinme committed:
Fixing a query for consistency

anavinahar committed:
Clarification

Based on CSS case questions

mmacy committed:
Merge pull request #10 from neilpeterson/nepetes-acs-updates

copy update

v-thepet committed:
Merge pull request #17993 from WilliamAntonRohm/w-dla-quota-toc

added "ADLA Quota Limits" article to TOC

Commit:
Merge pull request #17996 from anavinahar/patch-24

Clarification

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-concurrency

HeidiSteen committed:
code formatting

swkrish committed:
Update active-directory-aadconnect-sso-faq.md

v-thepet committed:
Merge pull request #17987 from steveesp/vnet-optimal

Vnet optimal

swkrish committed:
Update active-directory-aadconnect-troubleshoot-sso.md

v-thepet committed:
Merge pull request #17904 from mmacy/acs-kube-split-dcosswarm-02

[acs] split k8s and dcos/swarm

nitinme committed:
removing blank space

Commit:
Merge pull request #16835 from burkeholland/patch-8

Add Bootstrap Form styles

v-thepet committed:
Merge pull request #16323 from HeidiSteen/heidist-concurrency

Azure Search: Concurrency article and TOC entry

v-thepet committed:
Merge pull request #17997 from nitinme/nitinme-hdi-happy-path

Fixing Spark queries for consistency

v-thepet committed:
Merge pull request #17334 from v-viinde/peoplecart

USER STORY 1037362 SaaS App Tutorial: Peoplecart

MightyPen committed:
Acrolinx - fix some legacy stuff.

Not trying to fix all the legacy stuff.

JeffGoldner committed:
Update azure-stack-sql-resource-provider-deploy.md

Fixed a few issues reported

v-thepet committed:
Merge pull request #17691 from swkrish/master

Added machine rollover and Edge/Safari support (DSSO)

Commit:
Merge pull request #17994 from MightyPen/patch-1

Clarify columnstore ix merely maintained at Standard tier

Commit:
Merge pull request #17999 from JeffGoldner/patch-1

Update azure-stack-sql-resource-provider-deploy.md

johnkemnetz committed:
Update monitoring-activity-log-schema.md

Commit:
typo

v-thepet committed:
Merge pull request #17933 from johnkemnetz/activity-log-schema

Add Activity log schema

Commit:
Merge pull request #18000 from stevestein/patch-2

typo

curtand committed:
first updated classic portal articles

v-stgreg committed:
Added note that Windows Phone 8.1 dev is not supported in VS 2017

mmacy committed:
[acs] link fix - file 1 of 6: container-service-faq

mmacy committed:
[acs] link fix - file 1 of 6: container-service-faq

mmacy committed:
[acs] link fix - file 2: container-service-scale

Commit:
Merge pull request #18002 from v-stgreg/mobile-engagement

Added note that Windows Phone 8.1 dev is not supported in VS 2017

mmacy committed:
[acs] link fix - file 3: container-service-security

mmacy committed:
[acs] link fix - file 4: container-service-connect

mmacy committed:
[acs] link fix - file 4: container-service-connect

genlin committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 61799

ramankumarlive committed:
fixed the wrong script for managed disks

ramankumarlive committed:
fixed formating issue

Commit:
Merge pull request #18006 from ramankumarlive/patch-2

fixed the wrong script for managed disks

GitHubber17 committed:
Merge pull request #17746 from genlin/61799

virtual-machines-classic-recovery-disks-portal.md

Commit:
Merge pull request #18007 from ramankumarlive/patch-3

fixed formating issue

GitHubber17 committed:
Merge pull request #18003 from mmacy/acs-link-fix-01

[acs] link fix for container-service includes

GitHubber17 committed:
Merge pull request #18001 from curtand/class0723

first updated classic portal articles

adegeo committed:
Updated .net article for framework 4.7

dominicbetts committed:
Review Pi basic samples

dominicbetts committed:
Review simulator samples

dominicbetts committed:
Review advanced samples

dominicbetts committed:
Review intro page

dominicbetts committed:
Review gateway kit samples

dominicbetts committed:
Review what are preconfigured solutions

dominicbetts committed:
Review predictive walkthrough

dominicbetts committed:
Review CF customize

anoobbacker committed:
Policy can't be renamed

Commit:
Merge pull request #18017 from anoobbacker/patch-8

Policy can't be renamed

bwren committed:
updates

bwren committed:
metadata

cychua committed:
Clarify group-based filtering and multi-forest

Commit:
Merge pull request #18021 from cychua/master

Clarify group-based filtering and multi-forest

rjagiewich committed:
Merge pull request #18013 from dominicbetts/freshness

Review IoT Suite articles

Lisaco88 committed:
Merge pull request #16275 from bwren/o365

Office 365 solutions

ponatara committed:
Update site-recovery-vmm-to-azure.md

adegeo committed:
updated for acro

ponatara committed:
Update site-recovery-vmm-to-azure.md

Lisaco88 committed:
Merge pull request #17742 from JPallavi/patch-22

Create backup-azure-log-analytics-data-model.md

Commit:
Merge pull request #17974 from mumian/hbaserefresh0724

test the streaming application

ponatara committed:
Update site-recovery-vmm-to-azure.md

CarlRabeler committed:
Update sql-database-resource-limits.md

Commit:
Merge pull request #18024 from ponatara/patch-2

Update site-recovery-vmm-to-azure.md

Commit:
Merge pull request #18026 from CarlRabeler/patch-302

Update sql-database-resource-limits.md

bwren committed:
typos

ponatara committed:
Update site-recovery-faq.md

Commit:
Merge pull request #18029 from ponatara/patch-3

Update site-recovery-faq.md

Commit:
Merge pull request #18030 from bwren/o365

Office 365 typos

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #18033 from MicrosoftDocs/FromPublicRepo

From public repo

Lisaco88 committed:
Merge pull request #18011 from Thraka/net

Updated .net article for framework 4.7
